responses/11-targets-debug.md

```diff
@@ -6,17 +6,21 @@ If you've written your own functions or scripts before, you may have run into th
 
 Breakpoints allow you to run a function (or script) up until the line of the breakpoint, and then the evaluation pauses. You are able to inspect all variables available at that point in the evaluation, and even step carefully forward one line at a time. It is out of scope of this exercise to go through exactly how to use debuggers, but they are powerful and helpful tools. It would be a good idea to read up on them if you haven't run into breakpoints yet. 
 
-In `scipiper`, you can't set a breakpoint in the "normal" way, which would be clicking on the line number after you sourced the script. Instead, you need to use the other method for debugging in R, which requires adding the function call `browser()` to the line where you'd like the function call to stop. 
+In `targets`, you can't set a breakpoint in the "normal" way, which would be clicking on the line number after you sourced the script. Instead, you need to use the other method for debugging in R, which requires adding the function call `browser()` to the line where you'd like the function call to stop and specifying an additional argument when you call `tar_make()`. 
 
 ---
 
-You have a working, albeit brittle, pipeline in your course repository. You can try it out with `scipiper::scmake()`. This pipeline has a number of things you'll work to fix later, but for now, it is a useful reference. The pipeline contains several functions which are defined in `.R` files. 
+You have a working, albeit brittle, pipeline in your course repository. You can try it out with `targets::tar_make()`. This pipeline has a number of things you'll work to fix later, but for now, it is a useful reference. The pipeline contains several functions which are defined in `.R` files. 
```

Comment:
Ah! I didn't even notice that. I suppose that was an issue with the previous course, as well. I think we should move the cloning instructions up. I wonder if it should be in an earlier step (perhaps the "overview" one?) so that it is set up way ahead of time and users can choose to follow along with the pipeline code on GitHub or locally.